plugins/com.google.cloud.tools.eclipse.sdk.ui/src/com/google/cloud/tools/eclipse/sdk/ui/MessageConsoleWriterOutputLineListener.java

```diff
@@ -22,6 +22,7 @@
 
 public class MessageConsoleWriterOutputLineListener implements ProcessOutputLineListener {
```

Comment:
Probably not for this PR, but it feels to me like we've got something seriously factored wrong here and elsewhere. We should not be plugging into the console to get the information we need. It ought to all come from appengine-plugins-core.